requirements/specifications/lifecycle/index.md

```diff
@@ -1,274 +1,365 @@
-state at one time.
+It is a platform decision as to how many apps may be in this state at
+once.
 
-Apps in this state **MUST** be visible.
+Apps in this state **MAY** be visible.
 
-Apps in this state **MUST** receive RCU key presses that they have
+Apps in this state **MAY** receive RCU key presses that they have
 permissions for.
 
-Apps in this state **MUST** have access to the audio-video decoder.
+Apps in this state **MUST** have access to the audio decoder if
+they have permission to the `xrn:firebolt:capability:decoder:audio`
+capability and the platform has an available software or hardware
+decoder.
+
+Apps in this state **MUST** have access to the video decoder if
+they have permission to the `xrn:firebolt:capability:decoder:video`
+capability and the platform has an available software or hardware
+decoder.
+
+Apps in this state **MUST** have access to the graphics compositor.
 
 Apps in this state **SHOULD NOT** have performance negatively impacted
 by other processes on the device.
 
+Apps in this state **SHOULD** use less than XXX megabytes of RAM.
+
+**TODO**: do we want tiers of apps? per-app config?
+
 When an app transitions to this state, the platform **MUST** dispatch
-the Lifecycle.onForeground notification with the current state and
+the `Lifecycle.onActive` notification with the current state and
 previous states.
 
-### 2.4. Background
-
-This state allows an app to be present as the secondary experience to
-the user, e.g. when a system settings UI is overlayed on top of the app.
 
-Different platforms **MAY** support differing numbers of background
-apps. See [Background App
-Requirements](https://comcastcorp.sharepoint.com/:w:/r/sites/hqppa/Shared%20Documents/Global%20App%20Platform/Firebolt/Requirements/Core/Lifecycle/Background%20App%20Requirements.docx?d=w185d9112bcd44376bb0635dad816919b&csf=1&web=1&e=cjmuSJ)
-for more info.
+### 2.4. Suspended
 
-Apps in this state **MUST** either be partially visible **OR** have
-access to an audio decoder, e.g. an app playing music in the background.
+This state allows an app to remain in memory and consume fewer resources.
 
-Apps in this state **SHOULD NOT** receive most RCU key presses, although
-platforms **MAY** forward certain keys to the app, e.g. play/pause keys.
+Suspended apps can execute code, but are not [Presented](./presentation.md) to the user, do not receive
+any input from RCUs, and cannot use the video decoder.
 
-Apps in this state **MAY** have access to the audio-video decoder. How
-many decoders are available is a decision for each device based on
-resources. See [Background App
-Requirements](https://comcastcorp.sharepoint.com/:w:/r/sites/hqppa/Shared%20Documents/Global%20App%20Platform/Firebolt/Requirements/Core/Lifecycle/Background%20App%20Requirements.docx?d=w185d9112bcd44376bb0635dad816919b&csf=1&web=1&e=cjmuSJ)
-for more info.
+Apps in this state **MUST NOT** have access to the graphics compositor.
```

Comment:
Apps in this state **MUST NOT** have associated Graphics Surface created/allocated. Need to destroy, deallocated that before moving to this state as part of suspend() call